docs/reference/run.md

```diff
@@ -617,7 +617,13 @@ and require killing system processes to free memory.
 By default, a container's kernel can swap out a percentage of anonymous pages.
 To set this percentage for a container, specify a `--memory-swappiness` value
 between 0 and 100. A value of 0 turns off anonymous page swapping. A value of
-100 sets all anonymous pages as swappable.
+100 sets all anonymous pages as swappable. By default, if you are not using
+`--memory-swapiness`, memory swappiness value will be inherited from parent.
+
+Note that since 0 is an effective value, so we take -1 as the default value
```

Comment:
I would leave this paragraph out.  We say valid values are between 0-100 why do we need to mention -1 case?